### PR TITLE
feat: add ipfs link next to transaction viewer

### PR DESCRIPTION
**Pull request by daywiss**
### Issues
There was a request to include ipfs link next to transaction viewer

### Changes 
This will render an ipfs link whenever it exists, next to the transaction title.
image

### How to test
*_(Explain how the changes can be tested, including any required setup steps)_*

1. view any proposal with a transaction viewer, see the ipfs link exists, click it. 

### To-Do
*_(List any outstanding tasks be addressed before or after this PR is merged)_*

- [ ] 

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [ ] I have tested my changes on a custom domain
- [ ] I have run end-to-end tests `yarn cypress:test:e2e`, and they have passed

### Additional notes or considerations
*_(Include any other relevant information or context that may be helpful for the reviewer)_*

### CODE REVIEW

**Comment by pemulis** [comment]
I like the placement! But would reword this from View Strategy to View Details. Hopefully that will make it clear enough for proposers and voters that they can find full transaction details, like the address of a token being transferred, by clicking the link. It will also make it easier for validators to find the full proposal details to decide whether or not to dispute an on-chain proposal.

**Review by samuveth — APPROVED** [review]
tACK
